azerothcore/azerothcore-wotlk
# feat(DB/Translations) Imported german translations from GTDB

##### CHANGES PROPOSED:
I imported from [GTDB](https://github.com/gpn39f/GTDB) ~and [GCMDB](https://github.com/Xadras/GCMDB/)~ several string translated into german for deDE client as @Endeffect-AT suggested.

I imported the followings table and related data, I removed the duplicates data:

- creature_text_locale (452)
- creature_template_locale (only 40, the others 22k are already imported)
- gossip_menu_option_locale (2924)
- item_template_locale (165)
- npc_text_locale (5395)
- page_text_locale(28)
- points_of_interest_locale (0) -> AC has already all data
- trinity_string (904)
- quest_template_locale (1), these quests are already translated for deDE client.
- ~quest_request_items_locale (3082)~,  ~quest_request_items_locale (648) **(GCMDB)**~
- ~quest_offer_reward_locale (5680)~, ~quest_offer_reward_locale (2936) **(GCMDB)**~

##### ISSUES ADDRESSED:
- Closes #2346 

##### HOW TO TEST THE CHANGES:
 - Import the SQL file and check the translations
 - Check that the others locales are untouched after the importing


##### Target branch(es):
